### ISSUE TITLE
How to connect to Remote Hbase from spark/SHC

### ISSUE THREAD

**Issue by AbdullahServantOfAllah** [hortonworks-spark/shc#227]
The example available connect to localhost by default.
But If I have seperate Spark cluster and Hbase cluster respectively. Then how to specify and connect to remote hb ase. Please give sample configuration and sample code as applicable.

**Comment by weiqingy** [comment]
@AbdullahServantOfAllah The configurations for the cases of that Spark and Hbase are in different cluster is the same as the cases of both Spark and Hbase are in the same cluster. You needs to copy the hbase-site.xml from the hbase cluster to the folder `spark/conf/`.

**Comment by AbdullahServantOfAllah** [comment]
@weiqingy ,
Thanks for the info, just copying hbase-site.xml is sufficient right? I dont need to specify in code(or configuration) the hbase's zookeeper url and port right?

**Comment by weiqingy** [comment]
No, you don't need to specify anything in the code. The configurations in the hbase-site.xml should be enough for SHC/Spark to identify your Hbase cluster. Spark reads the configurations specified in hbase-site.xml before starting running jobs.

**Comment by mukesh14j** [comment]
Passing hbase-site.xml as classpath will also work?
